Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1477,14 +1478,14 @@ void StyleComputer::transform_box_type_if_needed(StyleProperties& style, DOM::El
             //        to do that specifically. For now, we simply check for "inline-flex" and convert
             //        that to "flex".
             if (display.is_flex_inside())
-                style.set_property(CSS::PropertyID::Display, IdentifierStyleValue::create(CSS::ValueID::Flex));
+                style.set_property(CSS::PropertyID::Display, DisplayStyleValue::create({ CSS::Display::Outside::Block, CSS::Display::Inside::Flex }));
             else
-                style.set_property(CSS::PropertyID::Display, IdentifierStyleValue::create(CSS::ValueID::Block));
+                style.set_property(CSS::PropertyID::Display, DisplayStyleValue::create({ CSS::Display::Outside::Block, CSS::Display::Inside::Block }));
```

Comment:
Again, `Display::from_short()` would work well here.

Reply:
I would ignore that if its ok because in the next commit we need to change only the outside value and thats why we have the full constructor.